packages/cinterop/src/darwin/kotlin/io/realm/internal/interop/RealmInterop.kt

```diff
@@ -932,10 +954,11 @@ actual object RealmInterop {
                 for (i in 0 until num_headers.toInt()) {
                     headers?.get(i)?.let { header ->
                         headerMap[header.name!!.toKString()] = header.value!!.toKString()
-                    } ?: error("Header at index $i within range ${num_headers.toInt()} should not be null")
+                    }
+                        ?: error("Header at index $i within range ${num_headers.toInt()} should not be null")
```

Comment:
It is the IDE's suggestion 🤷

I usually try to avoid pushing reformatting of the whole file, but this was from an old _WIP_ commit, so did not go back and clean it up. I have given up on reasoning on this as I don't get the conventions. If we don't want to follow these, we should maybe reconfigure it so that we got assistance on fulfilling the rules.